bin/cron/summarize_census_data

```diff
@@ -1,71 +0,0 @@
-#!/usr/bin/env ruby
-
-# The map on code.org/yourschool is run off of a Google Fusion Table
-# (documentation: https://developers.google.com/maps/documentation/javascript/fusiontableslayer).
-# This script updates the fusion table using the REST API
-# (documentation: https://developers.google.com/fusiontables/docs/v2/using),
-# populating it with all rows from the census_summaries table for the appropriate school year.
-
-# After setting up authentication, provide the following CDO.* attributes:
-#   CDO.hoc_map_account: (JSON of a credential which can edit the FusionTables table)
-#   CDO.census_map_table_id: (the id of the fusion table)
-
-require_relative '../../dashboard/config/environment'
-require 'cdo/properties'
-
-require 'google/apis/fusiontables_v2'
-require 'googleauth'
-
-TABLE_ID = CDO.census_map_table_id.freeze
-
-def make_location(lat, long)
-  return nil if lat.nil? || long.nil?
-
-  "#{format('%.6f', lat)}, #{format('%.6f', long)}"
-end
-
-# Update summaries before updating the fusion table
-# If any single teaches_cs category in a given year changes by more than
-# 10% of the total number of schools then issue a Honeybadger notification.
-threshold = School.count * 0.1
-old_summary_counts = Census::CensusSummary.group(:school_year, :teaches_cs).count
-Census::CensusSummary.summarize_census_data
-new_summary_counts = Census::CensusSummary.group(:school_year, :teaches_cs).count
-
-unless old_summary_counts.select {|k, c| (c - new_summary_counts[k]).abs > threshold}.empty?
```

Comment:
I don't think we should delete this actually. We should replace the fusion tables code below with the Mapbox code.

Reply:
Or actually you could just revert d895dd077cfc3c6c47346c633b3be8dd5da7cf82.

Essentially there was a concern about the `CensusSummary` code running twice because it's actually an expensive DB operation, so I removed it from `update_census_mapbox`. It needs to be preserved somehow.

Reply:
Great catch! I updated the cron job. Please take a look